homeassistant/components/sensor/dsmr.py

```diff
@@ -26,18 +26,21 @@
 has been updated.
 """
 
+import asyncio
 import logging
 from datetime import timedelta
-import asyncio
-import voluptuous as vol
+
 import homeassistant.helpers.config_validation as cv
+import voluptuous as vol
 from homeassistant.components.sensor import PLATFORM_SCHEMA
-from homeassistant.const import CONF_PORT, EVENT_HOMEASSISTANT_STOP
+from homeassistant.const import CONF_PORT
 from homeassistant.helpers.entity import Entity
 
 DOMAIN = 'dsmr'
 
-REQUIREMENTS = ['dsmr-parser==0.3']
+REQUIREMENTS = [
+    'https://github.com/aequitas/dsmr_parser/archive/async_protocol.zip'
```

Comment:
Do you know if this will be merged upstream? Or are you planning on maintaining an async fork?

This also needs a version at the end `...zip#dsmr-parser==0.4` for example.